MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include README.md
 include src/VERSION
 include src/config/default.yml
-include src/data_hub/callback.js
\ No newline at end of file
+include src/data_hub/*.js
```

Comment:
we now include all JS files in the package

Reply:
Maybe we nest js under it's own dir?